content/geoip/updating-databases.mdx

```diff
@@ -129,6 +129,23 @@ servers can make HTTPS connections to the following hostname:
 
 - `mm-prod-geoip-databases.a2649acb697e2c09b632799562c076f2.r2.cloudflarestorage.com`
 
+For example, using the permalink for the GeoIP2 City CSV database, you can issue
+a `curl` command like the following:
+
+```cli
+curl -O -J -L -u YOUR_ACCOUNT_ID:YOUR_LICENSE_KEY 'https://download.maxmind.com/geoip/databases/GeoIP2-City-CSV/download?suffix=zip'
+```
+
+Or a `wget` command like the following:
+
+```cli
+wget --content-disposition --user=YOUR_ACCOUNT_ID --password=YOUR_LICENSE_KEY 'https://download.maxmind.com/geoip/databases/GeoIP2-City-CSV/download?suffix=zip'
+```
+
+Where `YOUR_ACCOUNT_ID` is a placeholder for your account’s actual account ID
+and `YOUR_LICENSE_KEY` is a placeholder for your account’s actual license key.
+[Learn more about license keys on our knowledge base](https://support.maxmind.com/hc/en-us/articles/4407116112539-Using-License-Keys).
+
```

Comment:
Maybe a line saying that these will save the database to a file called `GeoIP2-City-CSV_YYYYMMDD.zip` where YYYYMMDD is the date of the latest release would also make sense.